include/nlohmann/json.hpp

```diff
@@ -5886,6 +5983,8 @@ class basic_json
     {
         std::swap(m_type, other.m_type);
         std::swap(m_value, other.m_value);
+
+        set_parents();
```

Comment:
Need to call `other.set_parents();` too.

Reply:
Good catch!

Reply:
Fixed in b0d8628c498d1a2be3391a1949886843746b9878.